examples/ota-requestor-app/esp32/main/OTARequesterImpl.cpp

```diff
@@ -294,7 +294,7 @@ void ConnectToProvider(const char * ipAddress, uint32_t nodeId)
 
         PeerId peerID = fabric->GetPeerId();
         peerID.SetNodeId(providerNodeId);
-        operationalDeviceProxy = new chip::OperationalDeviceProxy(initParams, peerID);
+        operationalDeviceProxy = new chip::OperationalDeviceProxy(initParams, peerID, nullptr);
```

Comment:
add a comment on what nullptr means like `nullptr /* nodeResolutionData */ `.

Can we avoid needing to provide the nullptr here? have a default argument or separate constructor?

Same comment below: bare nullptr/true/false arguments are hard to read.